src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -1185,6 +1185,37 @@ class C
         }
 
         [Fact]
+        public void OwningSymbolCSharp()
+        {
+            const string source = @"
+class C
+{
+    public void UnFunkyMethod()
+    {
+        int x = 0;
+        int y = x;
+    }
+
+    public void FunkyMethod()
+    {
+        int x = 0;
+        int y = x;
+    }
+
+    public int FunkyField = 12;
+    public int UnFunkyField = 12;
```

Comment:
OK--I'll switch the name to "ContainingSymbol" for the individual node contexts. If we hadn't already shipped code block contexts with an OwningSymbol property, we might discuss changing that name too, but we did, so we won't.